source/classes.tex

```diff
@@ -2477,7 +2477,7 @@
 };
 
 Y a;
-int b = a;          // error, \tcode{a.operator X().operator int()} not tried
+int b = a;          // error: \tcode{a.operator X().operator int()} not tried
```

Comment:
The text after the colon here is not a description of the error. Maybe

// error: no viable conversion (`a.operator X().operator int()` not considered)

Reply:
Fixed.